src/utils/cold-start.ts

```diff
@@ -0,0 +1,27 @@
+let _didFunctionColdStart = true;
+
+let _isColdStartSet = false;
```

Comment:
This is probably inconsistent with how we write js in other repos, but I don't think this repo uses underscores for global variables anywhere else.